nautobot/ipam/querysets.py

```diff
@@ -1,9 +1,56 @@
 import uuid
 
+import netaddr
+from django.db.models import ExpressionWrapper, IntegerField, F, QuerySet, Q, UUIDField
+from django.db.models.expressions import RawSQL
+from django.db.models.functions import Coalesce, Length
+
+from nautobot.ipam.constants import IPV4_BYTE_LENGTH, IPV6_BYTE_LENGTH
 from nautobot.utilities.querysets import RestrictedQuerySet
 
 
-class PrefixQuerySet(RestrictedQuerySet):
+class NetworkQuerySet(QuerySet):
+    def net_equals(self, prefix):
```

Comment:
It would be great to have some unit tests directly exercising each of these functions - there may be something similar in the tests already?

Reply:
Will do